specification/devcenter/data-plane/Microsoft.DevCenter/preview/2023-01-01-preview/devbox.json

```diff
@@ -1101,7 +1101,7 @@
           "$ref": "#/definitions/HardwareProfile"
         },
         "hibernateSupport": {
-          "description": "Indicates whether hibernate is enabled/disabled or unknown.",
+          "description": "Indicates whether hibernate is enabled/disabled or unknown. (Feature in Preview)",
```

Comment:
Are these qualifications really needed? The API is in preview -- this is clear by the API version.